tools/topology/topology1/sof/pipe-drc-eq-volume-demux-playback.m4

```diff
@@ -44,17 +44,30 @@ C_CONTROLBYTES(MY_MULTIBAND_DRC_CTRL, PIPELINE_ID,
 	,
 	MULTIBAND_DRC_priv)
 
+# DRC Enable switch
+define(MY_MULTIBAND_DRC_ENABLE, concat(`multiband_drc_enable_', PIPELINE_ID))
+define(`CONTROL_NAME', `MY_MULTIBAND_DRC_ENABLE')
```

Comment:
I found the root cause of my topology build error. The previous patch lacks CONTROL_NAME definition for this switch mixer. According to m4/mixercontrol.m4:
https://github.com/thesofproject/sof/blob/main/tools/topology/topology1/m4/mixercontrol.m4#L48

CONTROL_NAME will be renamed if not given.
Now my topology has no build error.